src/main/java/com/faforever/client/chat/AbstractChatTabController.java

```diff
@@ -497,6 +511,34 @@ public void openPrivateMessageTab(String username) {
    * Called from JavaScript when user clicked a URL.
    */
   public void openUrl(String url) {
+    if (url.contains(replayBaseUrl)) {
+      //wrong querry
```

Comment:
What does that misspelled comment mean? ;) Turn it in a clean, helpful comment or, better, remove it and make the code explaining itself.